### PR TITLE
OIA-26: Expose intrinsic tag names as constants for TSS API

### PR DESCRIPTION
**Pull request by patrick-schweizer**
Moves the constants from OpenNMS to the to the Integration API.

### External References

* Integration API PR: https://github.com/OpenNMS/opennms-integration-api/pull/28
* JIRA (Issue Tracker): https://issues.opennms.org/browse/OIA-26

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1414,8 +1414,8 @@
     <powermockVersion>1.6.4</powermockVersion>
     <okhttpVersion>3.10.0</okhttpVersion>
     <okioVersion>1.14.0</okioVersion>
-    <opennmsApiVersion>0.4.1</opennmsApiVersion>
-    <opennmsApiVersionOsgi>0.4.1</opennmsApiVersionOsgi>
+    <opennmsApiVersion>0.4.2-SNAPSHOT</opennmsApiVersion>
```
Comment (fooker) [inline pom.xml:1417]: Currently, I have no clue how the release cycle for the integration layer works. Do we have to change this before we merge to `release-26` but after we released the integration API?

**Comment by fooker** [comment]
@patrick-schweizer Can you link to the according PR for the integration API?

**Comment by patrick-schweizer** [comment]
> @patrick-schweizer Can you link to the according PR for the integration API?

You mean the link this in the description of this PR? The link to  the pull request in the API layer is:  https://github.com/OpenNMS/opennms-integration-api/pull/28

File: pom.xml (hunk 1 above)
Comment (patrick-schweizer) [reply pom.xml:1417]: Me neither. Every new commit in the master of the Integration API will release a new snapshot build automatically. So OpenNMS will compile and work. But of course we should reference a fixed version of the Integration Api before we release a new Opennms version.

@RangerRick would you know how we should proceed here?

**Comment by fooker** [comment]
> > @patrick-schweizer Can you link to the according PR for the integration API?
> 
> You mean the link in the description of this PR? The link to the pull request in the API layer is: [OpenNMS/opennms-integration-api#28](https://github.com/OpenNMS/opennms-integration-api/pull/28)

Exactly. I totally missed it in the description. Sorry.

File: pom.xml (hunk 1 above)
Comment (patrick-schweizer) [reply pom.xml:1417]: Talked to @RangerRick : It is ok to merge now. We just need to make a new Integration API version before a release. He will have his release script check for that :-)

Comment (RangerRick) [reply pom.xml:1417]: yup, updated... https://github.com/OpenNMS/opennms-repo/commit/ee4b7b6d971a3e4d20bc683c862be55bdb0f6bac

**Pull request merged**